From 22bc9088801e5f626f3a6409318e1593b457f701 Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Wed, 18 Jul 2018 15:07:45 +0000
Subject: [PATCH] coverity fix [#def6]

libreport-2.9.5/src/gui-wizard-gtk/wizard.c:819:5: warning: 1st function call
argument is an uninitialized value

https://cov01.lab.eng.brq.redhat.com/el8-results/el8/libreport-2.9.5-2.el8+7/scan-results.html#def6

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>


(cherry picked from commit 79650f521dc70535bf731ae3139860aef3e301b0)
---
 src/gui-wizard-gtk/wizard.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gui-wizard-gtk/wizard.c b/src/gui-wizard-gtk/wizard.c
index 35c6fc3..b4b2c3d 100644
--- a/src/gui-wizard-gtk/wizard.c
+++ b/src/gui-wizard-gtk/wizard.c
@@ -813,7 +813,7 @@ static void tv_details_row_activated(
                         GtkTreeViewColumn *column,
                         gpointer           user_data)
 {
-    gchar *item_name;
+    gchar *item_name = NULL;
     struct problem_item *item = get_current_problem_item_or_NULL(tree_view, &item_name);
     if (!item || !(item->flags & CD_FLAG_TXT))
         goto ret;
-- 
2.21.0

